app/src/MapView.tsx

```diff
@@ -152,11 +152,11 @@ function getMaplibreStyle(
   if (localSprites) {
     style.sprite = `${location.protocol}//${location.host}/${flavorName}`;
   } else {
-    style.sprite = `https://protomaps.github.io/basemaps-assets/sprites/v4/${flavorName}`;
+    style.sprite = `https://wipfli.github.io/basemaps-assets/sprites/v4/${flavorName}`;
```

Comment:
This line should be reverted. I am using the Zawgyi One Regular font in my basemaps-assets fork. Not sure how that one is licensed. We can use a different Zawgyi font too. Will look if I find one with SIL Open Font License...